man/docker-top.1.md

```diff
@@ -11,8 +11,9 @@ CONTAINER [ps OPTIONS]
 
 # DESCRIPTION
 
-Display the running process of the container. ps-OPTION can be any of the
- options you would pass to a Linux ps command.
+Display the running process of the container. ps-OPTION can be any of the options you would pass to a Linux ps command.
+
+All displayed information are from host's point of view.
```

Comment:
@vdemeester  information `is` --- treated as singular in English --- unlike French and German.